incubator/vault/Chart.yaml

```
@@ -1,8 +1,8 @@
 apiVersion: v1
 description: A Helm chart for Vault, a tool for managing secrets
 name: vault
-version: 0.14.1
-appVersion: 0.11.2
+version: 0.14.2
+appVersion: 0.11.3
```

The app version didn't change in this case AFAIK.
```suggestion
appVersion: 0.11.2
```